libs/ui/src/components/NeoDropdown/NeoDropdown.scss

```diff
@@ -29,8 +29,8 @@ $dropdown-item-hover-background-color: #ffe5f3;
   &.is-active,
   &--active {
     @include ktheme() {
-      color: theme('text-color-inverse');
-      background: theme('background-color-inverse');
+      color: theme('text-color-inverse')!important;
+      background: theme('background-color-inverse')!important;
```

Comment:
is the !important really necessary?

Reply:
yeah, without it, hover states were getting applied, when it shouldnt